cli/getting-started/index.md

```diff
@@ -69,26 +69,24 @@ https://github.com/terramate-io/terramate/releases
 ```
 :::
 
-Other installation options [are available](../installation.md#alternative-installation-methods). When the installation completes, you can test it out by reading the current version:
+Other installation options [are available](../installation.md#alternative-installation-methods). When the installation is complete, you can test it out by reading the current version:
 
 ```sh
 $ terramate version
 ```
 
 ### Install Terraform or OpenTofu
 
-Next, install Terraform or OpenTofu which we will orchestrate using Terramate:
+Next, install Terraform or OpenTofu to orchestrate it using Terramate:
```

Comment:
```suggestion
Next, install Terraform or OpenTofu:
```

sounds weird, maybe we just ditch orchestration here?

Reply:
yes, I think it's not needed here.